Resources/config/routing_pjax.yml

```diff
@@ -147,7 +147,7 @@ admin_roleList:
         _controller: ezsystems.platformui.controller.role:listRolesAction
 
 admin_roleView:
-    path: /role/view/{roleId}
+    path: /role/view/{roleId}/{tabId}
```

Comment:
@dpobel Agree, and I did that at first, but couldn't get it to work. The link was built without the trailing slash when tabId was omitted, meaning that the route did not match. So I added a second route without the tabId parameter - don't remember the details, but that didn't work well either. A working example would be helpful :)